src/main/webapp/index.html

```diff
@@ -11,7 +11,7 @@
  <body>
     <div id="content">
       <div>
-        <h1 class="heading-font">HOUSEHOLD</h1>
```

Comment:
hmm, maybe call this "title"?

I assume this is the font that is specifically used for the title